Sources/PackageCollections/Providers/JSONPackageCollectionProvider.swift

```diff
@@ -215,7 +217,33 @@ struct JSONPackageCollectionProvider: PackageCollectionProvider {
 extension Model.Product {
     fileprivate init(from: JSONModel.Product, packageTargets: [Model.Target]) {
         let targets = packageTargets.filter { from.targets.map { $0.lowercased() }.contains($0.name.lowercased()) }
-        self = .init(name: from.name, type: from.type, targets: targets)
+        self = .init(name: from.name, type: .init(from: from.type), targets: targets)
+    }
+}
+
+extension PackageModel.ProductType {
+    fileprivate init(from: JSONModel.ProductType) {
+        switch from {
+        case .library(let libraryType):
+            self = .library(.init(from: libraryType))
+        case .executable:
+            self = .executable
+        case .test:
+            self = .test
+        }
+    }
+}
+
+extension PackageModel.ProductType.LibraryType {
+    fileprivate init(from: JSONModel.ProductType.LibraryType) {
+        switch from {
+        case .static:
+            self = .static
+        case .dynamic:
+            self = .dynamic
+        case .automatic:
+            self = .automatic
+        }
     }
```

Comment:
I assume we dont re-user the package model types for separation of concerns?

Reply:
We don't reuse it so we don't have to add `PackageModel` dependency